### PR TITLE
doc: fix 'Adding-new-projects-to-wrapdb' to be more readable [skip ci]

### PR DESCRIPTION
**Pull request by koponomarenko**
Signed-off-by: Kostiantyn Ponomarenko <konstantin.ponomarenko@gmail.com>

### CODE REVIEW

File: docs/markdown/Adding-new-projects-to-wrapdb.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Adding new projects to wrap
+# Adding new projects to WrapDB
 
-**If you don't have permissions to do something on this page, please
-  open issue against https://github.com/mesonbuild/wrapweb/issues to
-  let us know that you want to start new project.**
+If you don't have permissions to do something on this page, please
```
Comment (jpakkane) [inline docs/markdown/Adding-new-projects-to-wrapdb.md]: The "don't have permissions to do something on this page" is actually leftover from the time when this doc was hosted in the Github wiki. The whole bit should be removed as it is confusing and replace with something simpler like "If you would like to submit a new Wrap to the WrapDB, please file an issue on the wrapweb bug tracker".

Comment (koponomarenko) [reply docs/markdown/Adding-new-projects-to-wrapdb.md]: Thanks for the feedback. I will fix it.

**Comment by jpakkane** [comment]
ping

**Comment by koponomarenko** [comment]
sorry, just didn't have much time this week ...

**Comment by jpakkane** [comment]
Sure, no rush.

**Comment by jpakkane** [comment]
These changes were merged as part of a different PR so closing.

**Pull request closed without merging**